Makefile

```diff
@@ -48,6 +48,10 @@ acceptance_parallel_tests:
 	export IGNORE_SEQUENTIAL_DATA_SETUP=True; \
 	pipenv run python run_in_parallel.py --command_line_args="${TEST_ARGS}"
 
+acceptance_sequential_tests_all: setup
+	export IGNORE_SEQUENTIAL_DATA_SETUP=False; \
+	pipenv run python run_in_sequence.py --command_line_args="${TEST_ARGS}" --test_tags="~donotskipany"
+
```

Comment:
Unless overridden, `TEST_ARGS` will read `--stop`, so this will stop on fail - do you intend that?

If you do, it _might_ be worth taking the opportunity to add a task to not stop?

Reply:
Fail fast, was my thinking